src/web/lib/alreadyVisited.ts

```diff
@@ -0,0 +1,11 @@
+const AlreadyVisitedKey = 'gu.alreadyVisited';
+
+export const getAlreadyVisitedCount = (): number => {
+    const alreadyVisited = parseInt(localStorage.getItem(AlreadyVisitedKey) || "", 10);
+    return !Number.isNaN(alreadyVisited) ? alreadyVisited : 0;
+};
+
+export const incrementAlreadyVisited = () => {
+    const alreadyVisited = getAlreadyVisitedCount();
+    localStorage.setItem(AlreadyVisitedKey, `${alreadyVisited + 1}`);
+};
```

Comment:
Perhaps in a follow up PR, it would be good to wrap these attempts to access `localStorage` with a try catch. In certain situations (Safari in private mode, when out of space, and others) the call will fail and without a try catch it throws an unhandled error.